index.html

```diff
@@ -2,61 +2,190 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Discover stays to live, work or just relax</title>
     <link rel="shortcut icon" href="images/favicon.svg" type="image/x-icon">
     <link href="style.css" rel="stylesheet">
 </head>
 <body>
-<div class="offers-block">
-    <div class="offers-block-inner">
-        <div class="offers-block-header"><h3>What do we offer</h3><hr></div>
-        <div class="offers-cards">
-            <div class="offers-item">
-                <div class="offers-icon"><img src="images/support.svg" alt="Support 24/7"></div>
-                <div class="offers-text">Support 24/7</div>
-            </div>
-            <div class="offers-item">
-                <div class="offers-icon"><img src="images/communication.svg" alt="Communicate directly"></div>
-                <div class="offers-text">Communicate directly</div>
+<header class="header">
+    <div class="wrapper">
```

Comment:
try combine '.wrapper' and '.container' in one class and make it one element in html instead of two